### ISSUE TITLE
how to combine instant ngp and  mobile nerf ?

### ISSUE THREAD

**Issue by Jiakui** [google-research/jax3d#117]
Dear author, 

instant ngp has the advantage that it is very fast to train,  is it possbile to convert the trained weight of instant ngp to mobile nerf ? If so, can you give me some directions so that I can try it by myself?

Thanks so much !

**Comment by czq142857** [comment]
No. You need to train mobilenerf from scratch because the mesh vertices are optimized throughout the training.

**Comment by kim-jinuk** [comment]
See [https://github.com/kim-jinuk/hash_mobilenerf.git](url)
